Podfile

```diff
@@ -9,8 +9,8 @@ platform :ios, '13.0'
 
 def wordpresskit_pods
   pod 'Alamofire', '~> 4.8.0'
-  pod 'CocoaLumberjack', '~> 3.4'
-  pod 'WordPressShared', '~> 1.15-beta' # will use release and beta versions up to 2.0
+  # pod 'WordPressShared', '~> 1.15-beta' # will use release and beta versions up to 2.0
+  pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared.git', :branch => 'trunk'
```

Comment:
We'll need to update this to use `'~> 2.0'` or `'~> 2.0', '>- 2.0.0-beta.1'` now that [2.0.0-beta.1 has been shipped](https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/326).